lib/manager/composer/artifacts.ts

```diff
@@ -31,9 +31,13 @@ interface UserPass {
   password: string;
 }
 
+interface GitlabToken {
+  [key: string]: string | string[];
+}
+
 interface AuthJson {
   'github-oauth'?: Record<string, string>;
-  'gitlab-token'?: Record<string, string>;
+  'gitlab-token'?: GitlabToken;
```

Comment:
```suggestion
  'gitlab-token'?: Record<string, string | string[]>;
```

🤗

Reply:
Thanks for the review :) 

It can't be `Record<string, string | string>` because the token will have two keys. The goal is to get a json such 
```
{"gitlab-token": {"gitlab.my-domain.com":"TOKEN", "gitlab-domains":["gitlab.my-domain.com"]}}
```

That's why I went with the `GitlabToken` interface :)

Reply:
```ts
interface GitlabToken {
  [key: string]: string | string[];
}
```

this is equal to 
```ts
type GitlabToken = Record<string, string | string[]>;
```

and `Record<string, string | string[]>`  is defined as 
```ts
type Record<K extends string, T> = {
  [P in K]: T
}
```

Reply:
Thanks a lot for the explanation!!
I updated the PR to remove the `GitlabToken` interface to use `Record<string, string | string[]>` instead